src/components/ToolbarSearch/ToolbarSearch.js

```diff
@@ -60,9 +60,11 @@ export default class ToolbarSearch extends Component {
     return (
       <ClickListener onClickOutside={this.handleClickOutside}>
         <div className={searchClasses} role={role}>
-          <label htmlFor={id} className="bx--label">
-            {labelText}
-          </label>
+          {labelText && (
```

Comment:
Is this a potential required prop since it provides the `htmlFor` attribute on the label for the `input`? Might fail a11y checks otherwise.

Reply:
We could go either way. I think it really depends on whether you want to force service teams to implement the labels or leave it up to them to handle whether or not to implement proper accessibility standards.